metaci/cumulusci/models.py

```diff
@@ -75,29 +119,38 @@ def get_absolute_url(self):
 
     @property
     def days(self):
-        return self.get_org_config().days
+        return self._get_org_config().days
 
     @property
     def days_alive(self):
-        return self.get_org_config().days_alive
+        return self._get_org_config().days_alive
 
     def get_org_config(self):
-        dx_local_dir = os.path.join(os.path.expanduser('~'), '.sfdx')
-        filename = os.path.join(dx_local_dir, '{}.json'.format(self.username))
-        with open(filename, 'w') as f:
-            f.write(self.json_dx)
+        return self._get_org_config()
 
+    def _get_org_config(self):
         org_config = json.loads(self.json)
         org_config['date_created'] = parse_datetime(org_config['date_created'])
         return ScratchOrgConfig(org_config, self.org.name)
 
+    def get_jwt_based_session(self):
+        return jwt_session('https://test.salesforce.com', self.username)
```

Comment:
Might be clearer to use a sandbox flag instead of passing the whole URL?

Reply:
yes. yes it would be :)